Doc/library/os.rst

```diff
@@ -3280,16 +3280,20 @@ written in Python, such as a mail server's external command delivery program.
    subprocesses.
 
 
-.. function:: register_at_fork(func, when)
+.. function:: register_at_fork(*, before=None, after_in_parent=None, \
+                               after_in_child=None)
 
-   Register *func* as a function to be executed when a new child process
-   is forked.  *when* is a string specifying at which point the function is
-   called and can take the following values:
+   Register callables to be executed when a new child process is forked
+   using :func:`os.fork`.  The parameters are optional and keyword-only.
+   Each specifies a different call point.
 
-   * *"before"* means the function is called before forking a child process;
-   * *"parent"* means the function is called from the parent process after
-     forking a child process;
-   * *"child"* means the function is called from the child process.
+   * *before* is a function called before forking a child process.
+   * *after_in_parent* is a function called from the parent process
+     after forking a child process.
+   * *after_in_child* is a function called from the child process
+     if control is expected to return to the Python interpreter.  A
```

Comment:
Is "if control is expected to return to the Python interpreter" related to other arguments? `_posixsubprocess.fork_exec()` doesn't call any of handlers if preexec_fn is None.

And the same question about the documentation of `PyOS_AfterFork_Child()` vs other functions.